app/services/id_token_builder.rb

```diff
@@ -38,6 +38,7 @@ def jwt_payload
   def id_token_claims
     {
       acr: acr,
+      vot: vot,
```

Comment:
also, I think we should make sure to add `vtm` if we add `vot` also ([spec link](https://openid.net/specs/openid-igov-openid-connect-1_0.html#RFC8485:~:text=REQUIRED%20if%20vot%20is%20provided.%20The%20trustmark%20URI%20as%20specified%20in%20Vectors%20of%20Trust%20%5BRFC8485%5D%20.%20See%20Vectors%20of%20Trust%20(Section%203.5)%20for%20more%20details.))

and to be clear, I think we should have both claims behind a feature flag to start with, so that we can quiet/soft launch this feature

and ideally, I think we would only only return the VOT claims if the clients requested VOT in the first place